### PR TITLE
BLD: Bump python support version 3.9

### PR DESCRIPTION
**Pull request by AnirudhDagar**
Follow up to https://github.com/d2l-ai/d2l-en/pull/2235/.

### CODE REVIEW

**Comment by AnirudhDagar** [comment]
@astonzhang I'm curious why was the latest API machine translations added [here](a3a2fc6ae991c6d6b4ebe3e46a74aec4adfa36f8) when d2l-zh is still following r0.17. This seems to be wrong, instead the machine translations should have used the r0.17 branch as base. I'm not sure why it was never caught before or maybe I'm wrong.

But it seems that the pdf building fails here due to that.

File: Jenkinsfile [hunk 1]
```diff
@@ -24,6 +24,7 @@ stage("Build and Publish") {
 
       sh label: "Sanity Check", script: """set -ex
       conda activate ${ENV_NAME}
+      d2lbook clear
```
Comment (astonzhang) [inline Jenkinsfile:27]:
```suggestion
```

File: chapter_installation/index.md [hunk 2]
```diff
@@ -8,19 +8,22 @@
...
-如果你使用macOS，假设你的Python版本是3.8（我们的测试版本），你将下载名称包含字符串“MacOSX”的bash脚本，并执行以下操作：
+如果你使用macOS，假设你的Python版本是3.9（我们的测试版本），你将下载名称包含字符串“MacOSX”的bash脚本，并执行以下操作：
 
 ```bash
 # 文件名可能会更改
-sh Miniconda3-py38_4.10.3-MacOSX-x86_64.sh -b
+# Intel Macs
+sh Miniconda3-py39_4.12.0-MacOSX-x86_64.sh -b
+# M1 Macs
+sh Miniconda3-py39_4.12.0-MacOSX-arm64.sh -b
```
Comment (astonzhang) [inline chapter_installation/index.md:18]:
```suggestion
sh Miniconda3-py39_4.12.0-MacOSX-x86_64.sh -b
```

File: chapter_installation/index.md [hunk 3]
```diff
@@ -8,19 +8,19 @@
 最简单的方法就是安装依赖Python 3.x的[Miniconda](https://conda.io/en/latest/miniconda.html)。
 如果已安装conda，则可以跳过以下步骤。访问Miniconda网站，根据Python3.x版本确定适合你的系统的版本。
 
-如果你使用macOS，假设你的Python版本是3.8（我们的测试版本），你将下载名称包含字符串“MacOSX”的bash脚本，并执行以下操作：
+如果你使用macOS，假设你的Python版本是3.9（我们的测试版本），你将下载名称包含字符串“MacOSX”的bash脚本，并执行以下操作：
 
 ```bash
 # 文件名可能会更改
```
Comment (astonzhang) [inline chapter_installation/index.md:14]:
```suggestion
# 以Intel处理器为例，文件名可能会更改
```

**Pull request merged**